master_server/database/redis.py

```diff
@@ -63,6 +63,7 @@ async def server_online(self, session_key, server_ip, server_port, info):
         await self._redis.set(f"ms-server-id:{session_key}", server_id, ex=TTL)
 
         info["game_type"] = 1  # Public
+        info["connection_type"] = 2  # Direct-IP
```

Comment:
Magic numbers

Reply:
With the comments there, I am fine with this for now. In the end, we need to solve https://github.com/OpenTTD/master-server/issues/43 which would allow access to the enums defining this. Similar for the others, ofc :)

A ticket so we don't forget ;)